eth2near/eth_rpc_client/src/beacon_rpc_client.rs

```diff
@@ -606,6 +606,26 @@ impl BeaconRPCClient {
         ))?
     }
 
+    pub fn get_left_non_empty_slot(
```

Comment:
is this code used?

Reply:
fixed https://github.com/aurora-is-near/rainbow-bridge/pull/883/commits/230ccb9835d31805895490cf6744d24ec47dba11